CHANGELOG.md

```diff
@@ -86,6 +89,17 @@
 
   The old uppercase settings `CONTEXT_BEHAVIOR` and `TAG_FORMATTER` are deprecated and will be removed in v1.
 
+- The setting `reload_on_template_change` was renamed to
+  [`reload_on_file_change`](../settings#django_components.app_settings.ComponentsSettings#reload_on_file_change).
+  And now it triggers server reload when any file in `BASE_DIR` changes. The old name `reload_on_template_change`
```

Comment:
You mean "when any **component** file changes" here right?

Reply:
Looks like you were looking at an older commit, the latest wording is "...when **any file in the component dirs** change".

At first, before I decided to extract `get_component_dirs()` and `get_component_files()` out of `autodiscover()` and `TemplateLoader`,  I tried implementing it by watching all files in `BASE_DIR`.

I then later refactored it, so only the files within `COMPONENTS.dirs` and `COMPONENTS.app_dirs` are watched, but forgot to update the changelog.

Reply:
Yes, my fault.